python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -163,7 +163,10 @@ def __init__( self, sceneView ) :
 					with GafferUI.ListContainer( spacing = 20 ) :
 						self.__attributeGroups = { a : _AttributeGroup(a) for a in _registeredShaderAttributes() }
 
-		# Main visibility is managed by the parent layout, so we manage our visibility of our frame
+		# The on/off state of the Inspector is managed by setVisible, which also disables lazy updates when
+		# were turned off. We desire to hide ourselves when we have nothing to show, but still need background
```

Comment:
were -> we're

Reply:
I don't know what is wrong with my brain, sorry, fixed [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/540625f6b4452ad4f3dec615e43566cadf0b2bf8).